metadata_tools/hosts/GO_0xxx/gcp_index_startup.sh

```
@@ -0,0 +1,22 @@
+apt-get update -y
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)